src/app-gocardless/tests/bank-factory.spec.js

```diff
@@ -1,50 +1,21 @@
 import BankFactory from '../bank-factory.js';
-import MbankRetailBrexplpw from '../banks/mbank-retail-brexplpw.js';
-import SandboxfinanceSfin0000 from '../banks/sandboxfinance-sfin0000.js';
-import IngPlIngbplpw from '../banks/ing-pl-ingbplpw.js';
+import { banks } from '../bank-factory.js';
 import IntegrationBank from '../banks/integration-bank.js';
-import Belfius from '../banks/belfius_gkccbebb.js';
-import SpkMarburgBiedenkopfHeladef1mar from '../banks/spk-marburg-biedenkopf-heladef1mar.js';
 
 describe('BankFactory', () => {
-  it('should return MbankRetailBrexplpw when institutionId is mbank-retail-brexplpw', () => {
-    const institutionId = MbankRetailBrexplpw.institutionIds[0];
-    const result = BankFactory(institutionId);
+  for (const bank of banks) {
+    for (const institutionId of bank.institutionIds) {
+      it(`should return institutionId ${institutionId}`, () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind moving the loop to use `it.each(testCases)` syntax? I'm very opinionated against having loops, conditionals and other logic in unit tests.